geoval/core/mapping.py

```diff
@@ -382,12 +382,13 @@ def _draw_cartopy(self, proj_prop=None, vmin_polygons=None, vmax_polygons=None,
         else:
             self.pax.set_global()  # ensure global plot
         self.pax.coastlines()
-
+        import traceback
```

Comment:
I keep traceback printout for a while (really hard to debug without it). Will remove if you think it should not be in the final version of the code.